openml-lightgbm/lightgbm-builder/pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <groupId>com.feedzai.openml.lightgbm</groupId>
     <artifactId>lightgbm-lib</artifactId>
-    <version>fairgbm-v1.0.0</version>
+    <version>v0.1.0</version>
```

Comment:
I don't get it, how is fairgbm distinguished from lgbm now?

Reply:
the fairgbm repo now only contains FairGBM code, i.e., all tags refer to FairGBM code.

this is distinguished from standard lightgbm by the repo at `lightgbm.repo.url`

Reply:
do you think the `lightgbm.version` should be different from the `lightgbmlib.version` to distinguish fairgbm from lightgbm better?

Reply:
I don't think it should be a different version. I have trouble understanding why it's not a different artifact. So, openml will stop making lgbm available? Only fairgbm from now on? Or am I misunderstanding?

Reply:
hmm perhaps we should schedule a quick call to clarify this.

The fairgbm repo also makes LightGBM available, and we need to point to the `v0.1.0` tag instead of the `fairgbm-v1.0.0` tag. However, the internal maven repository can have any tag it wants, perhaps we can maintain the tag for the maven artifact as prefixed by fairgbm.